internal/backend/sessions/sessionworkflows/metrics.go

```diff
@@ -10,11 +10,15 @@ var (
 	sessionsCompletedCounter metric.Int64Counter
 	sessionsErroredCounter   metric.Int64Counter
 	sessionsStoppedCounter   metric.Int64Counter
+
+	// NOTE: this is session time from AK perspective. Maybe we could ask temporal for better metrics?
+	sessionDurationHistogram metric.Int64Histogram
```

Comment:
That's a valid point
Maybe we should report when session workflow is done and not the actual session code

Wdyt?

Reply:
That's ther reason why I added a NOTE :) 
I think that for now it's OK, but we should think how to get a better metric.
btw, we are measuring the session workflow (in AK). what I would like to measure is the actual time spent in workers to run the session.

so, IMO it's good for now. I won't try to invent anything better before we are trying to work with a dashboard and see what's missing

Reply:
we decided to use AK's time, so I'm resolving this